project 1 html-css/Index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="style.css" rel="stylesheet"/>
+</head>
+
+
+<body>
+    <!-- header bar -->
+    <header class="top-bar">
+        <section class="company-name">My Logo</section>
+        <nav class="nav-btn">
+            <img  src="Enabled.png" alt="">
+        </nav>
+    </header>
+
+    <section class="main-pg">
+
+        <div class="hey">
+        Hey There.
+        <br>
+        I'm Ahmad.
+        </div>
+
+    <div class="software">
+        I'm a Software Developer
+    </div>
+
+    <p class="discription">
+        I can help you build a product , feature
+        <br>
+        or website Look through some of my
+        <br> work and experience! If you like what you
+        <br> see and have a project you need coded,
+        <br> 
+        don't hesitate to contact me.
+    </p>
+    
+
+        <ul>
+            <li class="git" ><img src="git logo.png" alt=""></li>
+            <li class="in-logo" ><img src="in.png" alt=""></li>
+            <li class="rabbit" ><img src="rabbit.png" alt=""></li>
+            <li class="twitter" ><img src="twitter .png" alt=""></li>
+            <li class="m-logo" ><img src="m-logo.png" alt=""></li>
+        </ul>
+        <img class="bg-img" src="Illustration.png" alt="">
```

Comment:
Use CSS `background-imag`e for Decorative Visuals Instead of `<img>`
The `<img>` tag should be reserved for images that are meaningful to the content (like profile photos, portfolio pieces, etc.).

For background visuals or illustrations that are only for design purposes and not content, it’s best to use CSS’s `background-image` property on a container or the page section.

This keeps your HTML clean, improves accessibility (by avoiding unnecessary or empty alt attributes), and separates content from presentation.